com.unity.netcode.gameobjects/Tests/Editor/Messaging/MessageSendingTests.cs

```diff
@@ -43,21 +44,36 @@ public void Send(ulong clientId, NetworkDelivery delivery, FastBufferWriter batc
             }
         }
 
-        private class TestMessageProvider : IMessageProvider
+        private class TestMessageProvider : IMessageProvider, IDisposable
```

Comment:
I'm confused, where the `TestMessageProvider.Dispose()` method? I can't see it in this PR 👀

Reply:
public void TearDown()
        {
            m_TestMessageProvider.Dispose();
            m_MessagingSystem.Dispose();
        }

Reply:
oh my lord! I'm blind lol it's just a few lines below HAHA